### DISCUSSION TITLE
Disable gtk for opencv?

### DISCUSSION THREAD

**Discussion by contagon** [microsoft/vcpkg#41850] [Q&A] [answered]
I'm trying to add opencv to my project _without_ gtk support built in. 

According to [here](https://vcpkg.io/en/package/opencv), it appears that if I add opencv[core] it should only add opencv4[core] as a dependency, not opencv4[default-features]. However, running `vcpkg depend-info opencv[core]` clearly still lists opencv4[default-features] as a dependency still. 

Is there anyway to add opencv with opencv4[core] not opencv4[default-features] being added?

**Comment by dg0yt** [comment, marked as answer]
To control the features of *opencv4*, you have to make an explicit statement about *opencv4*.

~~~sh
$ ./vcpkg depend-info opencv[core] opencv4[core]
vcpkg-cmake: 
vcpkg-cmake-config: 
vcpkg-get-python-packages: 
zlib: vcpkg-cmake
opencv4: vcpkg-cmake, vcpkg-cmake-config, vcpkg-get-python-packages, zlib
opencv: opencv4
~~~

**Reply by contagon** [reply]
That did the trick! Not sure why I never came to that conclusion. Thanks!
